inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/listener/SuspendSortListener.java

```diff
@@ -59,7 +59,7 @@ public ListenerResult listen(WorkflowContext context) throws Exception {
                 InlongGroupExtInfo::getKeyName, InlongGroupExtInfo::getKeyValue));
         String sortExt = kvConf.get(InlongGroupSettings.SORT_PROPERTIES);
         if (StringUtils.isEmpty(sortExt)) {
-            String message = String.format("inlongGroupId:%s not add suspendProcess listener,sortProperties is empty",
+            String message = String.format("inlongGroupId:%s not add suspendProcess listener, sortProperties is empty",
```

Comment:
Change the message to `String.format("groupId: %s not add suspendProcess listener, as the sortProperties is empty"`.